change/@fluentui-react-datepicker-compat-29b1faa0-25ce-495b-ac31-4c60408a64e6.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 15 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28299&commitId=31fca144a6f480235d7d8ecbde45398b0ea30a8a&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1933 screenshots unchanged, 0 screenshots with different dimensions and 15 screenshots with visible difference.


  <details><summary><b>unknown</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.DateRange- day.default.chromium.png|189| Changed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|140| Changed|
|DatePicker Compat.DateRange- month - High Contrast.default.chromium.png|119| Changed|
|DatePicker Compat.DateRange- month.default.chromium.png|140| Changed|
|DatePicker Compat.DateRange- week.default.chromium.png|164| Changed|
|DatePicker Compat.DateRange- work week.default.chromium.png|164| Changed|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|119| Changed|
|DatePicker Compat.default - High Contrast.opened.chromium.png|143| Changed|
|DatePicker Compat.default.opened.chromium.png|164| Changed|
|DatePicker Compat.marked dates - Dark Mode.default.chromium.png|200| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|197| Changed|
|DatePicker Compat.marked dates.default.chromium.png|189| Changed|
|DatePicker Compat.showWeekNumbers - Dark Mode.default.chromium.png|200| Changed|
|DatePicker Compat.showWeekNumbers - High Contrast.default.chromium.png|197| Changed|
|DatePicker Compat.showWeekNumbers.default.chromium.png|189| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>